multimap/src/main/java/org/infinispan/multimap/impl/EmbeddedMultimapSortedSetCache.java

```diff
@@ -113,7 +114,12 @@ private CompletionStage<CacheEntry<K, Collection<SortedSetBucket.ScoredValue<V>>
     */
    public CompletionStage<SortedSet<SortedSetBucket.ScoredValue<V>>> getValue(K key) {
       requireNonNull(key, ERR_KEY_CAN_T_BE_NULL);
-      return cache.getAsync(key).thenApply(b -> b.getScoredEntries());
+      return cache.getAsync(key).thenApply(b -> {
+         if (b != null) {
+            return b.getScoredEntries();
+         }
+         return null;
```

Comment:
I think this should be an empty collection instead?

Reply:
not for me, because this is "getValue" and when a value does not exist in a k-v is null 
this allows to know that the key does not exist. for the "zrange" command, it will be diff in my opinion